packages/cmf-router/src/defaultRouterConfiguration.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { hashHistory } from 'react-router';
+import { routerMiddleware, routerReducer, syncHistoryWithStore } from 'react-router-redux';
+import omit from 'lodash/omit';
+import get from 'lodash/get';
+import UIRouter from './UIRouter';
+
+const cmfMiddleware = store => next => action => {
+	const config = action.cmf;
+	if (!config) {
+		return next(action);
+	}
+	if (config.routerPush || config.routerReplace) {
+		let route = config.routerPush || config.routerReplace;
+		if (typeof route === 'function') {
```

Comment:
Shouldn't we use cmf mechanism to solve function, instead of allowing to dispatch action containing function ?